tdrs-backend/tdpservice/parsers/tanf_parser.py

```diff
@@ -1,31 +1,23 @@
 """Transforms a TANF datafile into an search_index model."""
 
 import logging
-from tdpservice.search_indexes.models.tanf import TANF_T1  # , T2, T3, T4, T5, T6, T7, ParserLog
+from tdpservice.search_indexes.models import T1, T2, T3  # , T4, T5, T6, T7, ParserLog
 # from django.core.exceptions import ValidationError
 from .util import get_record_type
-from .schema_defs.tanf import t1_schema
+from .schema_defs.tanf import t1_schema, t2_schema, t3_schema
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
-def active_t1_parser(line, line_number):
-    """Parse line in datafile as active case data, T1 only."""
-    family_case_schema = t1_schema()
-    # create search_index model
-    t1 = TANF_T1()
+def active_parse(actual_line_length, min_line_length, schema, line, line_number, model):
```

Comment:
Nit: change "model" to category_instance or maybe something more readable. "model" is very generic